Tella/Data/Database/TellaDataBase.swift

```diff
@@ -35,6 +35,8 @@ class TellaDataBase : DataBase {
             case 4:
                 createUwaziEntityInstancesTable()
                 createUwaziEntityInstanceVaultFileTable()
+            case 5:
+                addRelationshipColumnToUwaziTemplate()
```

Comment:
we need to include `addRelationshipColumnToUwaziTemplate` to the same version `4` (the previous version)